test/test_shell_template_prefix_util.py

```diff
@@ -29,6 +29,10 @@ def test_get_packages():
     with TemporaryDirectory(prefix='test_colcon_') as prefix_path:
         prefix_path = Path(prefix_path)
 
+        # mock no packages in not merged install layout
```

Comment:
Please remove "mock" from the comment since there is no mock involved in the following block.

Same below.

Reply:
done in https://github.com/colcon/colcon-core/pull/179/commits/dce2207141bfee1aceecce9806effed2f0e98ee7